### ISSUE TITLE
renamehost function results ignored by console handlers

### ISSUE THREAD

**Issue by rhinoau** [librenms/librenms#12979]
When using renamehost.php or the lnms device:rename wrapper, the result from the renamehost function is ignored in certain situations.

Toastr alerts for UI success or failure rarely seem to work but look correctly defined.

One example seen in discord discussions is when an rrd directory already exists with the new name, but there is no associated device name for that directory - unsure of how likely this event is or the likely causes.  Perhaps a previous rename failure/interruption or transient permissions errors during the rename process, has been seen in the wild though.

**Example:**
```
$ mkdir rrd/10.12.10.141
$ ./renamehost.php 10.12.10.140 10.12.10.141
```
**Output:**
```
Renamed 10.12.10.140
```
**Result:**
1. no renaming actions occur - no change.
2. device log entry appears: `Renaming of 10.12.10.140 failed`

**Expected Output:** 

The function log entry/return string shown on console:
```
Renaming of 10.12.10.140 failed
```

**Analysis:**

The function returns an empty string on success, else a failure message:
https://github.com/librenms/librenms/blob/69397ea70ffdcdf9e4dfeddfd71252f322b78ab5/includes/functions.php#L264-L278

1. 'console' usage - renamehost.php ignores any result and assumes success:
https://github.com/librenms/librenms/blob/7c157c1577c7d7dceb01269114f100d08609b911/renamehost.php#L27-L28 - 
2. 'webui' usage - checks for this but the Toastr alerts never appear on any of my systems good or bad:
https://github.com/librenms/librenms/blob/2d752925b0b0b3308c74fd7885a60a7cb87ab3d2/includes/html/pages/device/edit/device.inc.php#L50-L54
3. 'api' usage - have not tested, but checks for return:
https://github.com/librenms/librenms/blob/e670097bf6d1e49a5ed5983eff4e9367c72f8a47/includes/html/api_functions.inc.php#L1849-L1853

**Thoughts:**
Am unsure why the Toastr notifications do not appear in any case, but it looks like they should work as intended, as would the API.

For console usage, if renamehosts.php assigned the result and did conditional status reporting it would probably be enough without having to retrofit all the other dependent parts.  Would this specific use case example benefit from specific detection and reporting in renamehost.php that a directory already exists, or just falling back to 'failed' and correctly reporting be enough?

**Comment by rhinoau** [comment]
PR #12980 adds this to the stated example:
```
$ mkdir rrd/10.12.10.141
$ ./renamehost.php 10.12.10.140 10.12.10.141
NOT renamed: Renaming of 10.12.10.140 failed
$ echo $?
1
```
```
$ ./renamehost.php 10.12.10.140 10.12.10.142
Renamed 10.12.10.140
$ echo $?
0
```

**Comment by rhinoau** [comment]
Fixed by #12980
